.github/r_code/pdflatex_local_manual.r

```diff
@@ -0,0 +1,52 @@
+#' Comment out lines that prevent pdf from being generated locally and produce pdf
+#' 
+#' Reads the tex file and comments out lines that prevent pdf from being
+#' generated locally and  produces pdf using tinytex::pdflatex() and then changes 
+#' the tex file back.
```

Comment:
It would be nice to know why those lines that prevent the pdf from being generated locally are even there? Why can it be built online without this complication?

Reply:
Okay, will add info to direct people back to issue [#135](https://github.com/nmfs-ost/ss3-doc/issues/135).